### PR TITLE
fix: migrate settings and MaskMessages

### PR DESCRIPTION
**Pull request by lelenei**
## Description

Closes mf-4327

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/theme/src/Theme/types.ts [hunk 1]
```diff
@@ -1,5 +1 @@
-export enum Appearance {
-    default = 'default',
-    light = 'light',
-    dark = 'dark',
-}
+export {}
```
Comment (guanbinrui) [inline packages/theme/src/Theme/types.ts:1]: Remove this file.

File: packages/mask/shared/legacy-settings/settings.ts [hunk 2]
```diff
@@ -1,71 +1,8 @@
...
-import { Appearance } from '@masknet/theme'
-import { LanguageOptions } from '@masknet/public-api'
-import { EnhanceableSite, ExtensionSite, updateLanguage, NetworkPluginID } from '@masknet/shared-base'
-import { BooleanPreference } from '@masknet/plugin-infra'
+import { Appearance } from '@masknet/public-api'
+import { createGlobalSettings } from '@masknet/shared-base'
 import { Sentry } from '@masknet/web3-telemetry'
-import { createGlobalSettings, createBulkSettings } from './createSettings.js'
 
 export const appearanceSettings = createGlobalSettings<Appearance>('appearance', Appearance.default)
```
Comment (guanbinrui) [inline packages/mask/shared/legacy-settings/settings.ts:37]: Why still keep this setting here?

**Comment by Jack-Works** [comment]
hold on. I need to review this before merging. if you can tell me the rational behind this change it would be better.

**Review by guanbinrui — DISMISSED** [review]

**Review by Jack-Works — APPROVED** [review]
I think this okay

**Pull request merged**